coremltools/models/neural_network/optimization_utils.py

```diff
@@ -172,7 +172,11 @@ def _get_nn_mappings(layers):
 
 def _optimize_nn(layers):
     type_map, output_map, input_map = _get_nn_mappings(layers)
-    bn_layers = conv_layers = ip_layers = bias_layers = scale_layers = []
+    bn_layers = []
```

Comment:
I just checked, it seems `a = b= []` is same as `a=[]` and `b=[]` in python 2 and 3. I am surprised that this changed the behavior.